twitter-api/twitter-stream-map/config.py

```diff
@@ -0,0 +1,4 @@
+CONSUMER_KEY = 'jsjuaKAgWz4zDK2cKnGXf5Y30'
```

Comment:
Recommendation: Remove these lines and change your secrets from apps.twitter.com :)